### PR TITLE
Update docs sig info

### PR DESCRIPTION
**Pull request by ran-huang**
Update Docs SIG README.md according to the [template](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-template):

- Update learning materials
- Add `Pickable issues` and `Issues for newbie`

### CODE REVIEW

File: special-interest-groups/sig-docs/README.md [hunk 1]
```diff
@@ -47,11 +47,29 @@ There is no regular meeting currently.
...
 - Guides
   - [Documentation contributing guide](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md)
+  - [Google developer documentation style guide](https://developers.google.cn/style/)
   - [中文文档贡献指南](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md)
+  - [PingCAP 中文技术文档风格指南](https://github.com/pingcap/docs-cn/blob/master/resources/pingcap-style-guide-zh.pdf)
 - Documentation
   - [Documentation](https://docs.pingcap.com/)
   - [中文文档](https://docs.pingcap.com/zh/)
+
+## Pickable issues
```
Comment (ran-huang) [inline special-interest-groups/sig-docs/README.md:61]: We have several repos related to Docs SIG. Not sure how to format this.

File: special-interest-groups/sig-docs/README.md [hunk 2]
```diff
@@ -47,11 +47,29 @@ There is no regular meeting currently.
...
+
+- [Issues in docs](https://github.com/pingcap/docs/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+sort%3Aupdated-desc)
```
Comment (ran-huang) [inline special-interest-groups/sig-docs/README.md:63]: The links filter out issues that already have assignees.

**Review by lilin90 — APPROVED** [review]
LGTM

**Review by yikeke — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**